JSDemos/shared/azure.file.system.ts

```diff
@@ -344,7 +344,7 @@ export class AzureGateway {
         return x;
       })
       .then(async (x) => {
-        if (x.status === 200) {
+        if (x.status === 200 || x.status === 201) {
```

Comment:
This range of ok codes looks strange. Make sure that this is sufficient. The rest is fine.

Reply:
You can see in Microsoft documentation topics that the only success codes we can receive are 200 and 201. 
201 - for the [PutBlob](https://learn.microsoft.com/en-us/rest/api/storageservices/put-blob?tabs=azure-ad#status-code) request, when file is created
200 for the rest operations